data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,848 @@
+	"velocity scale" 0.4
+	
+effect "ballistic smoke"
+	sprite "effect/smoke"
+		"scale" 0.1
+		"no repeat"
+		"frame rate" 10
+	"lifetime" 30
+	"random spin" 6
+	"random angle" 360
+	"random velocity" 0.5
+	"velocity scale" 0.01
+
+outfit "Astuit Battery"
+	category "Turrets"
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 764000
+	thumbnail "outfit/astuit"
+	"mass" 28
+	"outfit space" -28
+	"weapon capacity" -28
+	"turret mounts" -1
+	weapon
+		sprite "projectile/astuit"
+		"hardpoint sprite" "hardpoint/astuit"
+		"hardpoint offset" 8
+		sound "astuit"
+		"hit effect" "astuit hit"
+		"inaccuracy" 5
+		"turret turn" 2.2
+		"velocity" 8
+		"lifetime" 44
+		"reload" 6
+		"firing energy" 13.2
+		"firing heat" 34
+		"shield damage" 8.8
+		"hull damage" 9.4
+	description "The Gegno Scin are the first Gegno to advance plasma technology to the point where it can be used in combat. Asuit Batteries work very similar to human Energy Blasters, firing small condensed plasma projectiles at high rates of fire. With 4 blasters on a turreted mount, it produces a constant stream of damage upon a target."
+	description `	Although each shot does not hit very hard, consistent fire adds up - but so does energy usage.`
+
+effect "astuit hit"
+	sprite "effect/ast hit"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 6
+	"velocity scale" .001
+
+
+outfit "Acuit Artillery"
+	category "Turrets"
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+	cost 1406000
+	thumbnail "outfit/acuit"
+	"mass" 81
+	"outfit space" -81
+	"weapon capacity" -81
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/acuit"
+			"random start frame"
+		"hardpoint sprite" "hardpoint/acuit"
+		"hardpoint offset" 14
+		"hit effect" "acuit hit"
+		sound "acuit"
+		"inaccuracy" 3
+		"turret turn" 1.3
+		"velocity" 18
+		"lifetime" 60
+		"reload" 56
+		"burst reload" 10
+		"burst count" 1
+		"firing energy" 130
+		"firing heat" 204
+		"shield damage" 170
+		"hull damage" 290
+		"hit force" 40
+	description `With a better grasp on plasma and energy weaponry, the Gegno Scin developed a larger blaster weapon to mount on the Feldspar Carrier as a counter to the Gegno Vi's huge cannon weaponry. The Acuit Artillery Cannon outranges most weapons in Gegno space, and has a rate of fire higher than most other weapons, making it a very efficient bombardment weapon from afar.`
+
+effect "acuit hit"
+	sprite "effect/acuit hit"
+		"frame rate" 18
+		"no repeat"
+	"lifetime" 18
+	"velocity scale" -0.004
+
+
+outfit "Guile Pulse Laser"
+	category "Guns"
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+	cost 109000
+	"thumbnail" "outfit/guile pulse laser"
+	"mass" 15
+	"outfit space" -15
+	"weapon capacity" -15
+	"gun ports" -1
+	weapon
+		"sprite" "projectile/guile pulse laser"
+			"frame rate" 6
+		sound "guile pulse"
+		"hit effect" "heavy laser impact"
+		"inaccuracy" 0.2
+		"velocity" 150
+		"lifetime" 1
+		"reload" 6
+		"burst count" 3
+		"burst reload" 1
+		"firing energy" 3.7
```

Comment:
An Energy Blaster consumes 29 energy/second while being 5 tons and dealing 53/33 shield/hull damage.
This consumes 37 energy/second at 15 tons and 124/202 shield/hull damage, so the energy usage really isn't as high as the description would suggest.